File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.lua
n = io.read("*number")
for j=1,n do
    v = io.read("*number")
    div=0
    if(v==2) then
        print(v.." eh primo")
    elseif(v==1) then
        print(v.." nao eh primo")
    elseif(v%2==0) then
        print(v.." nao eh primo")
    else
        for i=3,(v-1),2 do
            if(v%i==0) then
                div=1
                break
            end
        end
        if(div==0) then
            print(v.." eh primo")
        else
            print(v.." nao eh primo")
        end
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1165.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        int n, v, i, j;
        n = ler.nextInt();
        for(j=0;j<n;j++){
            v = ler.nextInt();
            int div=0;
            if(v==2){
                System.out.printf("%d eh primo\n", v);
            } else if(v==1){
                System.out.printf("%d nao eh primo\n", v);
            } else if(v%2==0){
                System.out.printf("%d nao eh primo\n", v);
            } else{
                for(i=3;i<v;i+=2){
                    if(v%i==0){
                        div=1;
                        break;
                    }
                }
                if(div==0){
                    System.out.printf("%d eh primo\n", v);
                } else{
                    System.out.printf("%d nao eh primo\n", v);
                }
            }
        }
    }
}